scripts/install-nfsserver.sh

```diff
@@ -54,11 +72,13 @@ exportfs
 ########################################
 # Tune NFS
 ########################################
-cores=$(grep processor /proc/cpuinfo | wc -l)
-nfs_proc=$(($cores * 4))
-replace="s/#RPCNFSDCOUNT=16/RPCNFSDCOUNT=$nfs_proc/g"
-sed -i -e "$replace" /etc/sysconfig/nfs
-grep RPCNFSDCOUNT /etc/sysconfig/nfs
+if is_centos7; then
```

Comment:
you should still increase the number of threads in centos8.  This is done by setting "threads" in the "[nfsd]" section in /etc/nfs.conf.